### ISSUE TITLE
kubeflow-profiles-operator failing integration tests track/1.7

### ISSUE THREAD

**Issue by i-chvets** [canonical/kubeflow-profiles-operator#130]
# Description

Discovered during running on pull request workflows when not related image retrieval script was checked in.
Integration tests are failing `ttox -e integration -- --model testing` in the workflow.
This PR has the logs/traces for failing tests https://github.com/canonical/kubeflow-profiles-operator/pull/129

Integration tests are failing due to `admission-webhook` `edge` not installing:
```
admission-webhook/0*  error     idle   10.1.115.73         hook failed: "install"
```

Need to review and address failing integration tests.

**Comment by i-chvets** [comment]
Integration tests are not related to the image collection script.
admission-webhook that is used in tests is latest/edge which is most likely incompatible with track/1.7.

Two options:
- Close this as it is caused by not related PR (not functionality related, but rather maintenenace/security scanning PR)
- FIx integration tests in track/1.7 to work with latest admission-webhook

**Comment by i-chvets** [comment]
Tests are not addressed as part of container images retrieval work. Closing.
